cdap-common/src/main/java/co/cask/cdap/common/twill/AbortOnTimeoutEventHandler.java

```diff
@@ -78,14 +78,14 @@ public void initialize(EventHandlerContext context) {
   public TimeoutAction launchTimeout(Iterable<TimeoutEvent> timeoutEvents) {
     long now = System.currentTimeMillis();
     for (TimeoutEvent event : timeoutEvents) {
-      LOG.info("Requested {} containers for runnable {}, only got {} after {} ms.",
+      LOG.warn("Requested {} containers for runnable {}, only got {} after {} ms.",
                event.getExpectedInstances(), event.getRunnableName(),
                event.getActualInstances(), System.currentTimeMillis() - event.getRequestTime());
 
       boolean pass = abortIfNotFull ? event.getActualInstances() == event.getExpectedInstances()
                                     : event.getActualInstances() != 0;
       if (!pass && (now - event.getRequestTime()) > abortTime) {
-        LOG.info("No containers for {}. Abort the application.", event.getRunnableName());
+        LOG.error("No containers for {}. Abort the application.", event.getRunnableName());
```

Comment:
Also log the application name and run id. Same for the log statement above.

Reply:
how do I get app name and run id? I don't see it in in event

Reply:
It is in the `EventHandlerContext`. You wrote that in Twill, remember?